static/js/tour.js

```diff
@@ -118,29 +103,48 @@ function startTour() {
+                using the two text boxes. The default value is 1.0x10^-5."
+            },
+            {
+                element: "#superPopContainer",
+                title: "Select super population of individual(s) in your VCF file",
+                content: "This will be used to perform LD clumping on your data (see the About \
+                page). If you are unsure about the super population of the individual(s) in your file, \
+                choose the one you believe to be the most accurate."
+            },
+            {
+                element: "#ldContainer",
+                title: "Select LD Clumping Type",
+                content: "Choose whether LD clumping should be performed sample-wide or by individual. For \
+                now, leave it at \"sample-wide clumping.\""
+            },
+            {
+                element: "#mafContainer",
+                title: "Select MAF population",
+                content: "This will be used to impute genotypes missing from your input data. Choose the \
+                population most representative of your input file for best results."
+            },
+            {
+                element: "#mafThreshContainer",
+                title: "Select MAF Threshold",
+                content: "This will filter out SNPs below the specified MAF threshold."
             },
             {
                 element: "#fileType",
                 title: "Select output format (pt 1)",
-                content: "Choose between TSV or JSON output format."
+                content: "Choose between TSV and JSON output format."
             },
             {
                 element: "#fileFormat",
                 title: "Select output format (pt 2)",
-                content: "Choose either a condensed results version or a full results version. The condensed version displays \
-                the polygenic risk scores for each sample for each trait/study combination. The full version additionally gives \
-                information for each sample on which SNPs contribute to the trait, which are protective against the trait, and \
-                which are neutral or have unknown contributions."
-            },
-            {
-                element: "#mafCohort",
-                title: "Select population for MAF",
-                content: "Select the population to use allele frequencies from."
-
+                content: "Choose either condensed results version or a full results version. The condensed version displays \
+                each study on a separate line with the sample polygenic risk scores as columns. The full version displays each \
+                sample/study pair on a separate line and additionally gives each sample's percentile compared to the selected \
+                MAF population as well as bar (|) separated lists of the protective, risk, non risk, and high LD SNPs involved \
+                in the calculation of the sample's score."
```

Comment:
```suggestion
                content: "Choose either condensed results version or a full results version. The condensed version displays \
                each study on a separate line with the sample polygenic risk scores as columns. The full version in tsv format displays each \
                sample/study pair on a separate line and additionally gives each sample's percentile compared to the selected \
                MAF population as well as bar (|) separated lists of the protective, risk, non risk, and high LD SNPs involved \
                in the calculation of the sample's score. The json format includes each sample nested under the trait/study object."
```

Reply:
I made this change with minor differences.